android/src/com/mapswithme/maps/routing/RoutingController.java

```diff
@@ -250,6 +250,9 @@ public void run()
 
   public void attach(@NonNull Container container)
   {
+    if (mContainer != null)
```

Comment:
Не обнуляется и ладно, сборщику мусора он мешать не будет - а большего и не нужно же.

> On Apr 5, 2016, at 3:16 PM, Alexander Marchuk <notifications@github.com> wrote:
> 
> In android/src/com/mapswithme/maps/routing/RoutingController.java <https://github.com/mapsme/omim/pull/2738#discussion_r58527135>:
> 
> > @@ -250,6 +250,9 @@ public void run()
> >  
> >    public void attach(@NonNull Container container)
> >    {
> > +    if (mContainer != null)
> WeakRef не сразу обнуляется. Повсюду расставлены проверки на null. Ок, уберу выброс исключения.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/2738/files/9d1faa5c75e529cf55423ea2e6bbb082207d8e98#r58527135>